components/LabelText/LabelText.tsx

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Gridiron Survivor.
+// Licensed under the MIT License.
+
+"use client"
+
```

Comment:
This extra space here seems weird.  Please maintain consistency with other [client components].(https://github.com/LetsGetTechnical/gridiron-survivor/blob/develop/components/Nav/Nav.tsx)

Reply:
I never noticed this; I've always added the space personally. It will be adjusted.

Reply:
Done, will be reflected in next commit.